### PR TITLE
add robot:mecharm-270

### PR DESCRIPTION
**Pull request by anla-xu**
add robot:mecharm-270

### CODE REVIEW

File: _posts/2022-07-28-mecharm-270.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- 'Robot'
+- 'Robotics'
+- 'RaspberryPi'
+- 'Roboticarm'
+- 'Maker'
+- 'DIY'
+- 'M5Stack'
+twitter_text: Checkout the mecharm-270
+website: https://www.elephantrobotics.com/en/mecharm-cn/
+wiki_homepage: https://www.elephantrobotics.com/en/mecharm-cn/
```
Comment (tfoote) [inline _posts/2022-07-28-mecharm-270.md]: This link should contain the ROS developer documentation for how to use the robot and not the product website. (Unless the docs are on the main website. But I don't find them there.)

**Review by tfoote — CHANGES_REQUESTED** [review]
Please provide a link to the developer documentation. 

I'd also suggest filling in a little bit more multimedia content or links to your description but that's not necessary.

**Comment by anla-xu** [comment]
A ros introduction URL has been changed, is it possible?

**Review by tfoote — APPROVED** [review]
Thanks for the update. That will work.

I still recommend slightly more graphically rich content but that's not a gating factor.

**Pull request merged**